Makefile

```diff
@@ -127,5 +127,6 @@ proto-lint:
 .PHONY: proto-lint
 
 proto-check-breaking:
-	@$(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=master
+	# we should turn this back on after our first release
+	# $(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=master
```

Comment:
Another way to do this is change to check against feature/cosmos-0.42 instead of master, but I think that's not useful and may be annoying.